events/eventservices/eventMapper_test.go

```diff
@@ -237,14 +238,14 @@ func msgChangedMessage(msgName string) string {
 
 func TestMapToFactomEvent(t *testing.T) {
```

Comment:
missing test for mapping AnchorEvent

Reply:
Still working on that, as of yet I still didn't get a live example through the live feed

Reply:
Done in next commit